azure-pipelines.yml

```diff
@@ -5,6 +5,24 @@
 jobs:
   - template: ./ci-jobs/functional_test.yml
 
+  - job: 'unitTests'
+    pool:
+      vmImage: 'ubuntu-16.04'
```

Comment:
why so old image?

Reply:
To find the cause of failing python versions... just tweaking config